### ISSUE TITLE
Add expensive recipes

### ISSUE THREAD

**Issue by Saintis** [doomeer/factorio#44]
Add the option to use the expensive recipes for items.

**Comment by Lava84flow** [comment]
yeah this would be pretty handy, it's a a pain to figure output to intake needs...

**Comment by JuicyJuuce** [comment]
@Saintis @Lava84flow 

Before @doomeer became active again, dwahler had a fork of this repository. I forked *that* and made the following patch a while ago. If someone want to make a pull request for it then go ahead, but it is based on a version from last May:

https://github.com/JuicyJuuce/factorio/blob/patch-1/recipes_expensive.ml
